msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1600592.java

```diff
@@ -79,25 +82,19 @@ public void handleUserInteraction() {
 
         authResult1.assertSuccess();
 
-        Thread.sleep(TimeUnit.MINUTES.toMillis(1));
-        Assert.assertTrue(mLabClient.resetPassword(username));
-
-        TestContext.getTestContext().getTestDevice().getSettings().forwardDeviceTimeForOneDay();
-
-        IAccount account = msalSdk.getAccount(mActivity,getConfigFileResourceId(),username);
+        // Silent with https://login.microsoftonline.com/common
         final MsalAuthTestParams silentParams = MsalAuthTestParams.builder()
                 .activity(mActivity)
                 .loginHint(username)
-                .authority(account.getAuthority())
+                .authority(getAuthority())
                 .resource(mScopes[0])
                 .msalConfigResourceId(getConfigFileResourceId())
                 .build();
 
         final MsalAuthResult authResult2 = msalSdk.acquireTokenSilent(silentParams, TokenRequestTimeout.MEDIUM);
 
-        // Should get a failure due to password reset.
         authResult2.assertFailure();
-        Assert.assertTrue(authResult2.getException() instanceof MsalUiRequiredException);
+        Assert.assertTrue(authResult2.getException() instanceof MsalServiceException);
```

Comment:
I think this test case is going to fail right now? that's what I recall from the manual testing done last week...if this is failing then you can add the ignore annotation to this one as well

Reply:
I got it to pass in automation, the silent call is returning an MsalServiceException about the application not being set up as multi tenant. This is the exception returned by the silent call.

AADSTS50194: Application 'a8c8bb57-55f3-4e95-9cf2-a3aac32f0d7a'(IDLABS_App_No_Admin_Consent) is not configured as a multi-tenant application. Usage of the /common endpoint is not supported for such applications created after '10/15/2018'. Use a tenant-specific endpoint or configure the application to be multi-tenant.
Trace ID: 79c48634-ce46-4953-bac9-e6bdffa43e00
Correlation ID: f7e03b76-ba82-4743-bb66-ca41e3a2bba2
Timestamp: 2022-06-10 04:18:35Z

Reply:
That's great. that means that something was wrong then in manual execution. cool.